service/operator/framework.go

```diff
@@ -0,0 +1,131 @@
+package operator
```

Comment:
Most of the removed lines are copy/pasted here + additional extra required by new version of the operatorkit. I find it clear to have this framework creation extracted to a file.